scripts/redact_cli_py/Pipfile.lock

```diff
@@ -403,12 +402,23 @@
             "markers": "python_version >= '3.6'",
             "version": "==2.5.0"
         },
-        "pypdfium": {
-            "hashes": [
-                "sha256:d890581c98417fad829a0312ab78d1e9ac6f0ff0722df3e472c75cbc8bd0eb8f"
+        "pypdfium2": {
+            "hashes": [
+                "sha256:11d7dc3d3bac2ac8ed5b2454acb3e6fa76c96c4a0eae13ab6a259672d36f25d9",
+                "sha256:2755a326bae555be8cb40ba4439be8bbf3080e49cd8a4f0781983d7a2bba3277",
+                "sha256:287d5292ad03a25d94d4772e292a9d573c0a6dee14b63b8dba8198eee33651cc",
+                "sha256:3b256b46872ba56a65f655c073d7a18814752c8a3d3ce837b993e134da0511eb",
+                "sha256:4438308ca6e32b4213e1099a630323a90e1a07d93f8c2e53dbe99c840ad99d33",
+                "sha256:4c0c49ac20bc5a30e70b059847aa6f17185637f75a55c647485b2fb7f92b4348",
+                "sha256:54ea53a035c98754c7ffcec84291ac6c1e52890b58409c450af8841ebab99f20",
+                "sha256:668cd1156c3308b46743c15d64482c448dfe35970efc7266f52d6dc2c5feefff",
+                "sha256:7c622df47543dd013bc7b7a0bc98db15182e5f0177e138a6be7c529028d10868",
+                "sha256:894a98f9f9d7d963ca4189c3c23a49715dae52e173acbb195276d46aef5396ea",
+                "sha256:ac720ad5072f4f30198f8830fdd8429493d61543159d98845b38b1d8210a8df3",
+                "sha256:dbedf7dbec23bf316ea7e6a466f130684d91536ea5c5b66dd237c77ad7e4ac63"
             ],
             "index": "pypi",
-            "version": "==0.0.15"
+            "version": "==3.1.0"
```

Comment:
This is the only relevant change to `Pipfile.lock` I wanted to do, but somehow `pipenv` updated lots of other parts on the way.